src/display/annotation_layer.js

```diff
@@ -2659,29 +2658,12 @@ class AnnotationLayer {
    * @memberof AnnotationLayer
    */
   static update(parameters) {
-    const { annotationCanvasMap, div, viewport } = parameters;
+    const { annotationCanvasMap, div } = parameters;
 
-    this.#setDimensions(div, viewport);
     this.#setAnnotationCanvasMap(div, annotationCanvasMap);
     div.hidden = false;
   }
 
-  /**
-   * @param {HTMLDivElement} div
-   * @param {PageViewport} viewport
-   */
-  static #setDimensions(div, { width, height, rotation }) {
```

Comment:
Here, and elsewhere, why are these methods being moved from the `src/`-folder and into the `web/`-folder? Won't that make the core PDF.js library less usable, assuming that we still care about non-Firefox use-cases here?

Reply:
I changed the dimensions to something using the page dimensions and `--scale-factor` so now they're constant whatever we do (zooming, rotating, ...). So I thought it'd make more sense to have this code where the div properties are initialized.